src/test/java/org/jabref/logic/citationstyle/CitationStyleGeneratorTest.java

```diff
@@ -66,80 +59,72 @@ void testHtmlFormat() {
         String style = CitationStyle.getDefault().getSource();
         CitationStyleOutputFormat format = CitationStyleOutputFormat.HTML;
 
-        String actualCitation = CitationStyleGenerator.generateCitation(entry, style, format);
-        assertEquals(expectedCitation, actualCitation);
-    }
-
-    @Test
-    void testRtfFormat() {
-        String expectedCitation = "[1]\\tab B. Smith, B. Jones, and J. Williams, \\uc0\\u8220{}Title of the test entry,\\uc0\\u8221{} {\\i{}BibTeX Journal}, vol. 34, no. 3, pp. 45\\uc0\\u8211{}67, Jul. 2016.\r\n";
-        BibEntry entry = TestEntry.getTestEntry();
-        String style = CitationStyle.getDefault().getSource();
-        CitationStyleOutputFormat format = CitationStyleOutputFormat.RTF;
-
-        String actualCitation = CitationStyleGenerator.generateCitation(entry, style, format);
+        String actualCitation = CitationStyleGenerator.generateCitation(entry, style, format, new BibDatabase());
         assertEquals(expectedCitation, actualCitation);
     }
 
     @Test
+    @Disabled("Currently citeproc does not handle number field correctly https://github.com/JabRef/jabref/issues/8372")
```

Comment:
This is strange, the test string reads correctly. Did your changes really make this test not passing?

Reply:
First of all, the whole class was disabled and my changes did not touch this:
This is what it outputs for the test entry: 
`[1]B. Smith, B. Jones, and J. Williams, “Title of the test entry,” BibTeX Journal, vol. 34, no. 3, Art. no. 3, Jul. 2016, doi: 10.1001/bla.blubb.
`